public/app/plugins/datasource/grafana-azure-monitor-datasource/components/QueryEditor/QueryTypeField.tsx

```diff
@@ -34,8 +44,8 @@ const QueryTypeField: React.FC<QueryTypeFieldProps> = ({ query, onQueryChange })
     <Field label="Service">
       <Select
         inputId="azure-monitor-query-type-field"
-        value={findOption(QUERY_TYPES, query.queryType)}
-        options={QUERY_TYPES}
+        value={findOption(queryTypes, query.queryType)}
+        options={queryTypes}
```

Comment:
it'd be worth to have some tests for this

Reply:
There are! 😭 https://github.com/grafana/grafana/blob/joshhunt/deprecate-app-insights-config/public/app/plugins/datasource/grafana-azure-monitor-datasource/components/QueryEditor/QueryEditor.test.tsx#L137-L183